.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ default_language_version:
 
```

Comment:
I ran, locally, `pre-commit autoupdate`, which generated this diff.  Then @philippemnoel told me to comment out the mintlify section and push all this.  So here we are.